src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourceValidator.cs

```diff
@@ -189,20 +175,13 @@ private static void SetAllowInsecureConnectionsProperty(PackageSource packageSou
             }
         }
 
-        private static PackageSource? FindByName(string name, List<PackageSource> packageSources)
+        private static PackageSource? FindById(string packageSourceId, List<PackageSource> packageSources)
```

Comment:
This has the same implementation as FindByName. 
I think this obfuscates the method behavior.

Reply:
> has the same implementation as FindByName.

There is no more FindByName. This PR renames it. I've reached out to speak offline when you're available.